op-bindings/bindings/addressmanager.go

```diff
@@ -156,11 +157,11 @@ func NewAddressManagerFilterer(address common.Address, filterer bind.ContractFil
 
 // bindAddressManager binds a generic wrapper to an already deployed contract.
 func bindAddressManager(address common.Address, caller bind.ContractCaller, transactor bind.ContractTransactor, filterer bind.ContractFilterer) (*bind.BoundContract, error) {
-	parsed, err := abi.JSON(strings.NewReader(AddressManagerABI))
```

Comment:
This looks like a new version of abigen was used locally, we should update in our build

Reply:
Hmm... I'll just use the old version of abigen for now.